<launch>
    <node name="joy_node" pkg="joy" type="joy_node" />
    <node name="tym_joycon_commander" pkg="tym_joycon_commander" type="tym_joycon_commander">
        <remap from="cmd_vel" to="/cmd_vel" />
        <param name="max_vel_double" value="1.0" />
        <param name="min_vel_double" value="0.1" />
        <param name="max_angle_double" value="1.0" />
        <param name="min_angle_double" value="0.2" />
        <param name="vel_step_double" value="0.1" />
        <param name="ang_step_double" value="0.1" />
    </node>
</launch>